jmclient/jmclient/wallet.py

```diff
@@ -1593,6 +1610,172 @@ def _get_mixdepth_from_path(self, path):
 
         return path[len(self._get_bip32_base_path())] - 2**31
 
+class FidelityBondMixin(object):
+    BIP32_TIMELOCK_ID = 2
+    BIP32_BURN_ID = 3
+
+    """
+    explaination of time number
+
+    incrementing time numbers (0, 1, 2, 3, 4...
+    will produce datetimes
+    suitable for timelocking  (1st january, 1st april, 1st july ....
+    this greatly reduces the number of possible timelock values
+    and is helpful for recovery of funds because the wallet can search
+    the only addresses corresponding to timenumbers which are far fewer
+
+    For example, if TIMENUMBER_UNIT = 1 (i.e. every time number is one month)
```

Comment:
Might be helpful to use an example other than '1' here: does TIMENUMBER_UNIT = 2 mean 6 values per year? I guess so, not sure.

Reply:
Noted, will do that